corehq/apps/api/domain_metadata.py

```diff
@@ -66,10 +66,27 @@ def dehydrate_calculated_properties(self, bundle):
                        .size(1)
                        .run()
                        .hits[0])
-            return {
+            base_properties = {
                 prop_name: es_data[prop_name]
                 for prop_name in es_data if prop_name[:3] == 'cp_'
             }
+            try:
+                audit_record = DomainAuditRecordEntry.objects.get(domain=domain.name)
+            except DomainAuditRecordEntry.DoesNotExist:
+                audit_record = None
+            extra_properties = [
+                "cp_n_downloads_custom_exports",
+                "cp_n_viewed_ucr_reports",
+                "cp_n_viewed_non_ucr_reports",
+                "cp_n_reports_created",
+                "cp_n_reports_edited",
+                "cp_n_saved_scheduled_reports",
+                "cp_n_click_app_deploy",
+                "cp_n_saved_app_changes",
+            ]
+            for prop in extra_properties:
+                base_properties.update({prop: getattr(audit_record, prop, 0)})
```

Comment:
why not just check if `audit_record` is None and skip this loop? Or even better put this in an `else` statement for the try block.

Reply:
@snopoke 
What if we will don't have a record ```DomainAuditRecordEntry``` for the specific domain. I think we should return 0s for it.

Reply:
True